src/app/stores/clients.ts

```diff
@@ -31,8 +31,13 @@ export const useClientsStore = defineStore('Clients', () => {
   const clients = ref<null | LocalClient[]>(null);
   const clientsPersist = ref<Record<string, ClientPersist>>({});
 
+  const searchParams = ref({
+    filter: '',
+  });
+
```

Comment:
you could export this directly and avoid the setSearchQuery function call

and just pass that directly as the v-model to the search field